src/controllers/arrays.js

```diff
@@ -0,0 +1,43 @@
+const {
+  getNthElement,
+  arrayToCSVString,
+  addToArray2,
+  elementsStartingWithAVowel,
+  removeNthElement2,
+} = require('../lib/arrays');
+
+const getElementNthIndex = (req, res) => {
+  const num = parseInt(req.params.number, 10);
+
+  res.status(200).json({ result: getNthElement(num, req.body.array) });
```

Comment:
This could go on one line, you dont need to create the `num` variable for it to only be used once:
```
  res.status(200).json({ result: getNthElement(parseInt(req.params.number, 10), req.body.array) });
```